Lambda-School-Labs/Labs8-RateMyDIY
# Feature/aws bucket setup

# Description

Added basic ability to upload files to our AWS s3 setup. Endpoint added '''/api/projects/image-upload'''
New dependencies needed, don't forget to '''yarn''' before testing.
multer multer-s3 aws-sdk

Fixes # (issue)

## Type of change


## Change status
- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

1. To test, run ```yarn''' then ''''yarn start```.
1. If you haven't already, check our slack project room for the "new .env variables" and add those to your .env
1. Open Postman and configure a test to POST to http://localhost:5000/api/projects/image-upload
1. Click "Body" and then "form-data" and add the key "image". Make sure to select "file": https://i.imgur.com/51xvlzC.png and chose a file to upload under the "value" column.
1. This will return an object with the URL of the image
```
{
    "imageUrl": "https://ratemydiy.s3.us-east-2.amazonaws.com/1542308547588"
}
```
# Checklist:
